domains/jirka086.json

```
@@ -4,7 +4,8 @@
     "email": "adamec08@seznam.cz"
   },
   "records": {
-    "CNAME": "web.vagonbrei.eu"
+    "CNAME": "web.vagonbrei.eu",
+    "TXT": "zoho-verification=zb51874936.zmverify.zoho.eu"
   },
   "proxied": false
```

you cant Combine CNAME with other records unless proxied
```suggestion
  "proxied": true
```

---

allright i can try to see if it works
if it breakes the website i will just delete the txt record and not have an email